### PR TITLE
Repo write commit metadata

### PR DESCRIPTION
**Pull request by dholms**
Return some optional commit metadata on repo write methods, namely commit CID & commit rev.

This is helpful metadata generally, but can be used in particular for something like managing read-after-write semantics by writing applications

### CODE REVIEW

File: lexicons/com/atproto/repo/applyWrites.json [hunk 1]
```diff
@@ -37,6 +37,16 @@
           }
         }
       },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "commitCid": { "type": "string", "format": "cid" },
+            "commitRev": { "type": "string" }
```
Comment (devinivy) [inline lexicons/com/atproto/repo/applyWrites.json:46]: If we want these to be required when either is present we could reshape it as `commit: { cid, rev }`.

File: packages/pds/src/api/com/atproto/repo/applyWrites.ts [hunk 2]
```diff
@@ -125,6 +125,14 @@ export default function (server: Server, ctx: AppContext) {
 
       await ctx.sequencer.sequenceCommit(did, commit, writes)
       await ctx.accountManager.updateRepoRoot(did, commit.cid, commit.rev)
+
+      return {
+        encoding: 'application/json',
+        body: {
+          commitCid: commit?.cid.toString(),
+          commitRev: commit?.rev,
```
Comment (devinivy) [inline packages/pds/src/api/com/atproto/repo/applyWrites.ts:133]:
```suggestion
          commitCid: commit.cid.toString(),
          commitRev: commit.rev,
```

**Review by devinivy — APPROVED** [review]
Lookin good!  Tossed out one idea but down with whatever you land on @dholms.

File: lexicons/com/atproto/repo/applyWrites.json (hunk 1 above)
Comment (dholms) [reply lexicons/com/atproto/repo/applyWrites.json]: yeah i was back and forth on this, but i dig 👍

**Pull request merged**